### ISSUE TITLE
ET ruleset support

### ISSUE THREAD

**Issue by NDietrich** [shirkdog/pulledpork3#1]
Enable EmergingThreats ruleset support in addition to Snort Rulesets
https://rules.emergingthreats.net/

**Comment by finchy** [comment]
Unless ET starts supporting Snort 3 with their ruleset, this can be closed.  The Snort 3 engine is not backwards compatible with Snort 2’s ruleset.

**Comment by NDietrich** [comment]
Thats right, I forgot that the Snort3 rule format is different enough from the Snort2 rule format that this isn't possible.  Thanks for flagging that @finchy.
I suppose we could add a feature to have PP run `snort2lua -c old.rules -r updated.rules` function on non-supported rulesets, but It's probably easier to downgrade this ticket to a 'someday project' to focus on more important things.
